### PR TITLE
A new landing page

### PR DESCRIPTION
**Pull request by Pomax**
New design based on https://github.com/mozilla/teach.mozilla.org/issues/1321

This branch is auto-deployed as http://ecks-rey-giggles-are-landing.herokuapp.com/

### CODE REVIEW

File: views/homepage/steps.html [hunk 1]
```diff
@@ -3,26 +3,47 @@
...
+            <img src="img/goggles-step-1.png">
+          </figure>
           <div class="description">
             <h2>Install Goggles</h2>
             <p>
               Make sure your bookmark bar is in view. Click and drag this
               button to your bookmark bar to install X-Ray Goggles.
             </p>
-            <a class="goggles-bookmark" href="javascript:"><span class="icon"></span>Drag and drop me in your bookmark bar</a>
+            <a class="goggles-bookmark" href="javascript:"><span class="dragme fa fa-arrows"></span>Drag and drop me in your bookmark bar</a>
```
Comment (Pomax) [inline views/homepage/steps.html]: turns out this *needs* to be the phrase "X-Ray Goggles", or the bookmark will read "Drag and drop m...", which we don't want.

File: public/stylesheets/homepage/style.less [hunk 2]
```diff
@@ -377,29 +366,34 @@ footer {
...
           color: white;
           text-align: center;
           text-decoration: none;
-
-          width: 15rem;
+          width: 17rem;
 
           &:hover {
-            background: #5BCAE3; // @skyblue:S:50%
+            background: #5BCAE3; // @blue:S:50%
```
Comment (mmmavis) [inline public/stylesheets/homepage/style.less]: less function awaits you :stuck_out_tongue_closed_eyes:

File: public/stylesheets/homepage/userbar.less [hunk 3]
```diff
@@ -21,7 +20,7 @@ body {
 }
 
 .navbar-goggles {
-  background-color: @purple;
+  background-color: #682060;
```
Comment (mmmavis) [inline public/stylesheets/homepage/userbar.less]: is it not `@purple` anymore? maybe it's worth turning it into a variable?

Comment (Pomax) [inline public/stylesheets/homepage/userbar.less]: there was never a @purple, that was from me landing a change to that file too soon.

**Comment by mmmavis** [comment]
arrrrr plusSsSss :heavy_plus_sign: :heavy_plus_sign:  :heavy_plus_sign:

**Pull request merged**